File: token-validation-spring/src/main/java/no/nav/security/token/support/spring/validation/interceptor/JwtTokenUnauthorizedException.java
package no.nav.security.token.support.spring.validation.interceptor;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@SuppressWarnings("serial")
@ResponseStatus(HttpStatus.UNAUTHORIZED)
public class JwtTokenUnauthorizedException extends RuntimeException {
	
	public JwtTokenUnauthorizedException(String msg) {
		super(msg);
	}

    public JwtTokenUnauthorizedException(Throwable cause) {
        super(cause);
    }
}

File: token-validation-spring/src/main/kotlin/no/nav/security/token/support/spring/validation/interceptor/JwtTokenUnauthorizedException.kt
package no.nav.security.token.support.spring.validation.interceptor

import org.springframework.http.HttpStatus.UNAUTHORIZED
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(UNAUTHORIZED)
class JwtTokenUnauthorizedException(msg: String? = null, cause: Throwable? = null): RuntimeException(msg,cause)